apps/mobile-wallet/src/features/buy/useOnramperUrl.ts

```diff
@@ -0,0 +1,26 @@
+import { AddressHash } from '@alephium/shared'
+import { useTheme } from 'styled-components/native'
+
+const ALPH_CODE = 'alph_alph'
+const ONRAMPER_CLIENT_UUID = 'pk_prod_01JJSAZY2D3XSZ614FKBC5EXYE'
+
+const useOnramperUrl = (receiveAddressHash: AddressHash) => {
+  const theme = useTheme()
+
+  // See https://docs.onramper.com/docs/supported-widget-parameters
+  return (
+    'https://buy.onramper.com/' +
+    '?mode=buy' +
+    `&onlyCryptos=${ALPH_CODE}` +
+    `&successRedirectUrl=${encodeURIComponent('https://alephium.org/banxa-callback/')}` +
```

Comment:
nit: we could rename the URL as `https://alephium.org/onramp-callback/`

Reply:
Done